pkg/components/components.go

```diff
@@ -163,3 +168,51 @@ func (c *components) getStandaloneComponents(scope string) []Component {
 	}
 	return standaloneComponents
 }
+
+// getDockerComposeComponents returns the list of all docker-compose Dapr components
+func (c *components) getDockerComposeComponents(scope string) []Component {
+	componentsDirectory := c.componentsPath
+	dockerComposeComponents := []Component{}
+	err := filepath.Walk(componentsDirectory, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			log.Printf("Failure accessing path %s: %v\n", path, err)
+			return err
+		}
+		if info.IsDir() && info.Name() != filepath.Base(componentsDirectory) {
+			return filepath.SkipDir
+		} else if !info.IsDir() && filepath.Ext(path) == ".yaml" {
+			content, err := ioutil.ReadFile(path)
```

Comment:
instead of `ioutil` which is deprecated use the equivalent functions from `os`

Reply:
I am fixing this now.